FWCore/ParameterSet/interface/ParameterSetConverter.h

```diff
@@ -50,6 +49,9 @@ namespace edm {
     ~ParameterSetConverter();
     ParameterSetIdConverter const& parameterSetIdConverter() const { return parameterSetIdConverter_; }
 
+    ParameterSetConverter(const ParameterSetConverter&) = delete;
+    ParameterSetConverter& operator=(const ParameterSetConverter&) = delete;
```

Comment:
I'd prefer the following order
```suggestion
    ParameterSetConverter(const ParameterSetConverter&) = delete;
    ParameterSetConverter& operator=(const ParameterSetConverter&) = delete;

    ParameterSetIdConverter const& parameterSetIdConverter() const { return parameterSetIdConverter_; }
```